### PR TITLE
Main

### PR DESCRIPTION
**Pull request by stevechude**

### CODE REVIEW

**Review by OyebisiJemil — COMMENTED** [review]
Once you appropriately ignore the node_modules file, files like the below won't be checked in.

[node_modules/.bin/acorn](https://github.com/CentBit-Internship/wallet-backend-a/pull/1/files#diff-7f1fb0ee3b4134e3e9ff84dbc39a2e9af273d01943de2ab1f17cbb01d90ab63d)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (OyebisiJemil) [inline .gitignore:1]: I could see you are trying to ignore node_modules here, but it is not working appropriately, could you find a way to do it appropriately?

**Review by OyebisiJemil — COMMENTED** [review]
I'll suggest you should try and explain your intent while making the PR, that could give the reviewer an idea of you you are doing.

**Comment by stevechude** [comment]
okay, i guess i'll just delete the node_modules and push the code again for
a new PR

**Pull request closed without merging**